deforestation_api/routers/healthcheck.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import APIRouter
+from fastapi import Response
+from healthcheck import HealthCheck
+from deforestation_api.settings import settings
+
+router = APIRouter(tags=["health"])
+
+
+@router.get("/health")
+def liveness() -> dict[str, str]:
+    return {"message": "Ok"}
+
+
+@router.get("/ready", tags=["health"])
+async def ready() -> Response:
+    health = HealthCheck()
```

Comment:
See flood-api. Refactor to initialize this healthcheck outside of the route-endpoint in order to accomplish caching.
Also, please include a healthcheck to see if the s3-files are available.

Reply:
There's an example of how to do this in flood-api

Reply:
Should be fixed now.